sites/main-site/src/content/special-interest-groups/regulatory/meetings.md

```diff
@@ -10,8 +10,7 @@ The Regulatory special interest group is going to Kick off on July 1st, 2024 at
 
 After the kick off, we will have regular meetups with a to-be-defined schedule that will be posted here, on LinkedIn and in the `#regulatory` Slack channel.
 
-|  Date | Speaker                                         | Event                                                   |
-| ----: | :---------------------------------------------- | ------------------------------------------------------- |
-| Jul 1 | Alex Peltzer (Boehringer Ingelheim Pharma GmbH) | [Kick off meeting](/events/2024/SIG_regulatory_kickoff) |
-| Sep 9 | Regulatory SIG | [Where do we stand and where do we want to go?](/events/2024/SIG_regulatory_2024-09-09.md) |
-
+|  Date | Speaker                                         | Event                                                                                      |
```

Comment:
Probably not - I think I merged in main at some point and this might have been picked up. It will likely be a conflict later so I'll deal with it then.